server/services/licenceService.js

```diff
@@ -350,6 +351,24 @@ module.exports = function createLicenceService(licenceClient) {
         return updatedLicence;
     }
 
+    async function rejectBass(licence, bookingId, bassRequested) {
+        const lastBassReferral = getIn(licence, ['bassReferral']);
+
+        if (!lastBassReferral) {
+            return licence;
+        }
+
+        const bassRejections = recordList({licence, path: ['bassRejections'], allowEmpty: true});
+        const licenceWithBassRejections = bassRejections.add({record: lastBassReferral});
+
+        const updatedLicence = replacePath(['bassReferral'], {bassRequest: {bassRequested}}, licenceWithBassRejections);
+
+        await licenceClient.updateLicence(bookingId, updatedLicence);
+
```

Comment:
I would just return licenceClient.updateLicence(bookingId, updatedLicence); and then this function doesn't need the async flag